commands/qemu-guest-tools/cmd.go

```diff
@@ -94,3 +129,28 @@ func (cmd) Execute(arguments map[string]interface{}) bool {
 
 	return true
 }
+
+func convertSimpleJSONTypes(val interface{}) interface{} {
```

Comment:
it's heartlessly stolen from `config/` package where I do the same thing... we should clean it up in some future for sure...